farmanager/__imageversion.txt

```diff
@@ -0,0 +1 @@
+3.0.4900
```

Comment:
This is really good idea but in relation to my previous response I would not use a project related tag (environment variable name) to keep the rest of the files (build, tag, test etc.) generic as much as possible.